docs/topics/data-analysis/data-analysis-work-with-api.md

```diff
@@ -1,16 +1,176 @@
 [//]: # (title: Retrieve data from web sources and APIs)
 
+Kotlin Notebook provides a powerful platform for accessing and manipulating data from various web sources and APIs.
+The [DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html) enables you to fetch and process data from online APIs.
+Whether you're extracting data, statistics, or other information, Kotlin Notebook simplifies these tasks by allowing you 
+to connect to APIs, parse the JSON responses, and transform them into structured data formats for comprehensive analysis 
```

Comment:
```
Whether you're extracting data, statistics, or other information, Kotlin Notebook simplifies these tasks by allowing you 
to connect to APIs, parse the JSON responses, and transform them into structured data formats for comprehensive analysis and visualization.
```
I somehow can't agree with this sentence. Kotlin Notebook simplifies accessing APIs because it is iterative (cost of an error is small), it can visualize every step, and it provides DataFrame with ability to generate properties for data access (you can call API without any prior knowledge and get fully typed structure as a result).
Things below are DataFrame responsibility 
connect to APIs, parse the JSON responses, and transform them into structured data